lib/database/session-manager.ts

```
@@ -0,0 +1,247 @@
+    };
+
+    if (deltas.messageCount) {
+      updates.messageCount = session.messageCount + deltas.messageCount;
+    }
+    
+    if (deltas.tokens) {
+      updates.totalTokens = session.totalTokens + deltas.tokens;
+    }
+    
+    if (deltas.cost) {
+      const currentCost = parseFloat(session.totalCost);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)